test-framework/keycloak-server/src/main/java/io/quarkus/test/keycloak/server/KeycloakContainer.java

```diff
@@ -10,7 +10,7 @@
 
 public class KeycloakContainer extends GenericContainer<KeycloakContainer> {
 
-    private static final String KEYCLOAK_VERSION = System.getProperty("keycloak.version");
+    private static final String KEYCLOAK_VERSION = System.getProperty("keycloak.server.version");
```

Comment:
I think I wouldn't do that in this PR.

If we want to change this, it warrants a proper separate discussion IMO as it will impact end users.